app/shared/src/image/NetworkImage.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 import Image from 'react-native-image-progress';
-import { connect } from '@kiwicom/react-native-app-redux';
+import { connect } from 'react-redux'; // TODO: Find out why this crashes when imported from @kiwicom/react-native-app-redux
```

Comment:
I don't want to merge this... :)

What problem is here? Is `connect` undefined? It may be a circular reference somewhere.